src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 900;
+}
+
+.data-form {
+  position: relative;
+  margin: 20px 0;
+}
+
+.big-search::after {
+  position: absolute;
+  content: "";
+  background-image: url(images/Search.svg);
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.small-search::after {
+  position: absolute;
+  content: "";
+  background-image: url(images/Search.svg);
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
```

Comment:
Set it only to `input` class, and override sizes for small input, you have 4 properties that are duplicated, DRY

Reply:
Or move duplicated properties to `input` class if you like to set sizes for both inputs separately